site/src/app/v2/trouver-un-club/components/club-finder/ClubFinder.tsx

```diff
@@ -66,22 +78,104 @@ const ClubFinder = ({ regions, activities, departments, isProVersion }: Props) =
       [SEARCH_QUERY_PARAMS.departmentCode]: searchParams.get(SEARCH_QUERY_PARAMS.departmentCode)
         ? `dep_code='${searchParams.get(SEARCH_QUERY_PARAMS.departmentCode)}'`
         : undefined,
+      [SEARCH_QUERY_PARAMS.distance]: undefined,
     }),
   });
 
-  const { clubName, regionCode, city, postalCode, activity, disability, offset } = clubParams;
+  const getParameterFromQuery = (searchQUeryParam: keyof typeof SEARCH_QUERY_PARAMS) => {
+    const param = searchParams && searchParams.get(SEARCH_QUERY_PARAMS[searchQUeryParam]);
+
+    if (searchQUeryParam === 'isShowingMapTab') {
+      if (isNaN(Number(param))) {
+        return undefined;
+      }
+      return param;
+    }
+  };
```

Comment:
Typo sur searchQ**U**eryParam et simplification suggérée
```suggestion
  const getParameterFromQuery = (searchQueryParam: keyof typeof SEARCH_QUERY_PARAMS) => {
    const param = searchParams && searchParams.get(SEARCH_QUERY_PARAMS[searchQueryParam]);

    if (searchQueryParam === 'isShowingMapTab') {
      return parseInt(param) === 1 ? param : undefined;
    }

    return undefined;
  };
```

Reply:
ca prend pas le cas ou param = 0 à priori.